pyunlocbox/tests/test_operators.py

```diff
@@ -20,45 +20,46 @@ def test_grad(self):
         mat2d = np.array([[2, 3, 0, 1], [22, 1, 4, 5]])
         mat3d = np.arange(1, 13).reshape(2, 2, 3).transpose((1, 2, 0))
         mat4d = np.arange(1, 25).reshape(2, 2, 2, 3).transpose((2, 3, 1, 0))
-        mat5d = np.arange(1, 49).reshape(2, 2, 3, 2, 2).transpose((3, 4, 2, 1, 0))
+        mat5d = np.arange(1, 49).reshape(2, 2, 3, 2, 2)
+        mat5d = mat5d.transpose((3, 4, 2, 1, 0))
 
         # 1D without weights.
-        dx = operators.grad(mat1d, dim=1)
+        dx = operators.grad1d(mat1d)
```

Comment:
Yes, but are those `grad1d(x)`, `grad2d(x)` wrappers useful ? For me `grad(x, dim=1)` is cleaner, but it's up to you. ;)